WORKSPACE

```diff
@@ -0,0 +1,69 @@
+new_git_repository(
```

Comment:
i don't know enough bazel: is this defining the googletest repo dependency for use in the BUILD? or is it supposed to define the benchmark package and this is a bad copy-paste?

Reply:
The former.

Here is an explanation of how the `new_git_repository` rule registers a git repository as a dependency:
https://docs.bazel.build/versions/master/be/workspace.html#git_repository

Reply:
But I believe you might have had question regarding the comment of copying the content of the BUILD file. The in-source comment is, to the best of my knowledge, stating the facts and rationale. Hopefully with a look at the `git_repository` rule that I pasted above, it clears out for you and potentially other reviewers.